db/seed/index.js

```diff
@@ -10,7 +10,7 @@ const {
     addCostumeToOrder,
     getCostumesByOrderId,
     getCostumeById
-} = require("../../index");
+} = require("../index");
```

Comment:
For future reference, this should probably be in a separate branch so you can land the fix before worrying about the feature